pkg/image/api/validation/validation.go

```diff
@@ -185,3 +183,45 @@ func ValidateImageStreamTagUpdate(newIST, oldIST *api.ImageStreamTag) fielderror
 
 	return result
 }
+
+func ValidateImageStreamImport(isi *api.ImageStreamImport) fielderrors.ValidationErrorList {
```

Comment:
> Not sure what you mean. Repository and Images together is intentionally valid, Images overrides Repository (just like it does today during import from the controller).

So both are valid, but you're ignoring one of them?  That's not intuitive.  Why would I ever bother with doing that?